extensions/interactions/FractionInput/directives/FractionInputValidationServiceSpec.js

```diff
@@ -139,11 +172,84 @@ describe('FractionInputValidationService', function() {
     function() {
       answerGroups[0].rules = [greaterThanMinusOne, equalsOneRule];
       var warnings = validatorService.getAllWarnings(
-        currentState, {}, answerGroups, goodDefaultOutcome);
+        currentState, customizationArgs, answerGroups,
+        goodDefaultOutcome);
       expect(warnings).toEqual([{
         type: WARNING_TYPES.ERROR,
         message: 'Rule 2 from answer group 1 will never be matched ' +
           'because it is made redundant by rule 1 from answer group 1.'
       }]);
     });
+
+  it('should catch redundant rules caused by exactly equals', function() {
+    answerGroups[0].rules = [exactlyEqualToOneAndNotInSimplestFormRule];
+    var warnings = validatorService.getAllWarnings(
+      currentState, customizationArgs, answerGroups,
+      goodDefaultOutcome);
+    expect(warnings).toEqual([{
+      type: WARNING_TYPES.ERROR,
+      message: 'Rule 1 from answer group 1 will never be matched ' +
+        'because it is not in simplest form.'
+    }]);
+  });
+
+
+  // ========================== NEW RULES =================================
```

Comment:
Not sure "new" will make any sense to readers of this code a year from now. Maybe use a different title?

Reply:
I went ahead and deleted this actually, it was supposed to be a temporary comment for my own benefit which I forgot to delete. Thanks!